test/e2e/latency.go

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"sync"
+	"time"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/cache"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/controller/framework"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util"
+	"k8s.io/kubernetes/pkg/util/sets"
+	"k8s.io/kubernetes/pkg/watch"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// This test suite can take a long time to run, so by default it is added to
+// the ginkgo.skip list (see driver.go).
+// To run this suite you must explicitly ask for it by setting the
+// -t/--test flag or ginkgo.focus flag.
```

Comment:
I don't think so -- I'll take a pass through and clean up.  Just wanted to
get a PR before the whole thing got stale.

On Mon, Sep 14, 2015 at 5:01 PM, Clayton Coleman <notifications@github.com>
wrote:

> In test/e2e/latency.go
> <https://github.com/kubernetes/kubernetes/pull/13932#discussion_r39446526>
> :
>
> > +	"k8s.io/kubernetes/pkg/controller/framework"
> > +	"k8s.io/kubernetes/pkg/fields"
> > +	"k8s.io/kubernetes/pkg/labels"
> > +	"k8s.io/kubernetes/pkg/runtime"
> > +	"k8s.io/kubernetes/pkg/util"
> > +	"k8s.io/kubernetes/pkg/util/sets"
> > +	"k8s.io/kubernetes/pkg/watch"
> > +
> > +	. "github.com/onsi/ginkgo"
> > +	. "github.com/onsi/gomega"
> > +)
> > +
> > +// This test suite can take a long time to run, so by default it is added to
> > +// the ginkgo.skip list (see driver.go).
> > +// To run this suite you must explicitly ask for it by setting the
> > +// -t/--test flag or ginkgo.focus flag.
>
> Is this comment true? I don't see a change to driver.go here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13932/files#r39446526>.
>